### PR TITLE
toggle selected mode

### PR DESCRIPTION
**Pull request by phoenixking25**
Toggle file format on switching json/yaml button in edit resource dialog.

### CODE REVIEW

File: src/app/frontend/common/dialogs/editresource/template.html [hunk 1]
```diff
@@ -30,7 +30,7 @@ <h2 mat-dialog-title>Edit a resource</h2>
     <mat-icon>info</mat-icon>
     <div>
       <span>This action is equivalent to:</span>
-      <code>kubectl apply -f &lt;spec.json&gt;</code>
+      <code>kubectl apply -f &lt;spec.{{selectedMode_}}&gt;</code>
```
Comment (jeefy) [inline src/app/frontend/common/dialogs/editresource/template.html]: When trying to run this locally I get:

```
[1] ERROR in src/app/frontend/common/dialogs/editresource/template.html(33,13): : Property 'selectedMode_' is private and only accessible within class 'EditResourceDialog'.
[1] ℹ ｢wdm｣: Failed to compile.
```

Which is true, `selectedMode_` is [declared as private](https://github.com/kubernetes/dashboard/blob/master/src/app/frontend/common/dialogs/editresource/dialog.ts#L33)

Comment (phoenixking25) [reply src/app/frontend/common/dialogs/editresource/template.html]: so should i make another public variable for this?

Comment (maciaszczykm) [reply src/app/frontend/common/dialogs/editresource/template.html]: You should change existing one to public and rename it to remove `_` from name.

Comment (phoenixking25) [reply src/app/frontend/common/dialogs/editresource/template.html]: ok thanks

**Comment by phoenixking25** [comment]
@maciaszczykm done

**Comment by maciaszczykm** [comment]
@phoenixking25 Have you tested it locally?

**Comment by phoenixking25** [comment]
yes

**Pull request merged**